pkg/gofr/container/sql_mock.go

```diff
@@ -0,0 +1,163 @@
+package container
+
+import (
+	"context"
+	gosql "database/sql"
```

Comment:
Instead of adding alias to the official library, let's add it to our GoFr Sql package.